app/Http/Controllers/PhotoController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Photo;
+use App\Room;
+use Response;
+
+class PhotoController extends Controller
+{
+    public function store($id, Request $request)
+    {
```

Comment:
What if the user will not upload image then click save?
I think it's better if you put image validation.

Reply:
done!